### ISSUE TITLE
Issue when getting to Grep?

### ISSUE THREAD

**Issue by aidenlt** [ChrisTitusTech/Debian-titus#85]
![Screenshot from 2023-04-15 22-25-50](https://user-images.githubusercontent.com/45278974/232264768-999cbfb9-4d9b-4663-bfe7-367f5013eb6d.png)

Was interested in deploying this in VirtManager and ran into this. Not knowledgeable enough to diagnose, but it was enough to halt my install.

**Comment by aidenlt** [comment]
Like I said, I'm not too knowledgeable on this but this commit did change root privileges around this point in the install, where Brave Browser gets installed.

https://github.com/ChrisTitusTech/Debian-titus/commit/c0ebc0edf39c5fd4fca2042fd3e1361888875089

**Comment by michaeldun56** [comment]
I'm having the same issue, same place aswell

**Comment by raman164** [comment]
yes, same error at same place for me. i am trying to install on my proxmox hypervisor platform. 
after rebooted, I managed to logged in, but with a weird looking screen. Just a desktop wallpaper and terminal with a high reso :(

**Comment by blusewill** [comment]
It's now fixed. I guess.
